package.json

```diff
@@ -6,9 +6,11 @@
   "workspaces": [
     "src/core/*",
     "src/core/components/*",
-    "packages/@guardian/*"
+    "packages/@guardian/*",
+		"scripts/analysis"
```

Comment:
```suggestion
        "scripts/analysis"
```

Reply:
Good catch 👾  Done in https://github.com/guardian/source/pull/972/commits/30305b6a42724d715c545e46c13e62450efff73a